scene/gui/color_picker.cpp

```diff
@@ -2206,6 +2324,14 @@ ColorPicker::ColorPicker() {
 	c_text->connect(SceneStringName(text_changed), callable_mp(this, &ColorPicker::_text_changed));
 	c_text->connect(SceneStringName(focus_exited), callable_mp(this, &ColorPicker::_html_focus_exit));
 
+	text_copy = memnew(Button);
+	hex_hbc->add_child(text_copy);
+	text_copy->set_icon_alignment(HORIZONTAL_ALIGNMENT_CENTER);
+	text_copy->set_tooltip_auto_translate_mode(AUTO_TRANSLATE_MODE_ALWAYS);
```

Comment:
What's the translate mode for?
EDIT:
I see you just copied it. It's not needed here.